documentation/modules/post/windows/manage/install_python.md

```diff
@@ -0,0 +1,80 @@
+This module downloads an embeddable Python3 interpreter onto the target file system, granting pentesters access to a lightweight Python interpreter. This module does not require administrative privileges or user interaction with installation prompts.
+### Creating A Testing Environment
+
+  This module has been tested against:
+
+1. Windows 10, 1903
+
+## Verification Steps
+
+  1. Start msfconsole
+  2. Get a Meterpreter session running as a low privilege user
+  3. Do: `use post/windows/manage/install_python`
+  4. Do: `set session #`
+  5. Optional Do: `set PYTHON_URL`
+  6. Optional Do: `set FILE_PATH`
+  7. Do: `run`
+
+
+## Options
+
+  **PYTHON_URL**
+
+  Specifies the URL used to download the Python embeddable zip file. Downloads Python version 3.8.2 by default.
+
+  **FILE_PATH**
+
+  Specifies the directory to place the Python embeddable zip file.
+  Places Python zip file in the current working directory by default.
+
+  **CLEANUP**
+
+  If true, this option will delete the Python zip file as well as its extracted contents. It will also terminate running processes with name 'python', as you cannot delete the Python interpreter if it is actively running.
+
+## Scenarios
+
+Get initial access: Create a Meterpreter exe using msfvenom, then transfer it to the target system via web server, SMB, etc. Execute the Meterpreter payload as a non-administrative user.
```

Comment:
Does it have to be a non-administrative user for some reason?

Reply:
It does not. I deleted that guidance.